.github/workflows/pull_request.yml

```diff
@@ -14,6 +14,51 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  kind_setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out forklift repository
+        uses: actions/checkout@v3
+
+      - name: Checkout forkliftci
+        uses: actions/checkout@v3
+        with:
+          repository: kubev2v/forkliftci
+          path: forkliftci
+
+      - name: Bazel cache
+        id: bazel-cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/bazel
+          key: ${{ runner.os }}-bazel-cache-${{ hashFiles('WORKSPACE') }}
+
+      - name: Add cwd to path for kubectl.
+        run: echo `pwd` >> $GITHUB_PATH
+
+      - name: Get kubectl
+        run: curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+      - run: chmod u+x kubectl
+
+      - name: Build and setup everything with bazel
+        run: |
+          cd forkliftci
+          FORKLIFT_DIR=${{ github.workspace }} ./build_and_setup_everything_bazel.sh
+          cd ..
+
+      - run: kubectl version
+
+      - run: kubectl wait deployment -n konveyor-forklift forklift-controller --for condition=Available=True --timeout=180s
+
+      - run: kubectl get pods -n konveyor-forklift
```

Comment:
This is an optional debug-step, we might not need it, once things are known to generally work. (It's not expensive, of course, but causes clutter.)

Reply:
if it shows the output somewhere, it won't harm.
what will happen if we drop this line and we just get the timeout in the line above? or better say - how we fail if the deployment fails?

Reply:
Good question! I don't know. It depends on the failure. The "get pods" can't fail, AFAIK, it can just return no pods.

Reply:
OK, than it also grants us a hint in such case. Can we leverage it to know if it's empty, fail with error?

Reply:
Yes, we can. For example:
     - run: echo "RUNNING_FORKLIFT_PODS=$(kubectl get pods -A | grep forklift | grep Running | wc | tr -s ' ' | cut -f 2 -d ' ')" >> $GITHUB_ENV
      - name: verify_four_pods
        if: ${{ env.RUNNING_FORKLIFT_PODS }} != "4"
        uses: actions/github-script@v3
        with:
          script: |
            core.setFailed('Wrong number of running forklift pods.')